airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -232,6 +238,38 @@ public WorkspaceRetentionConfig getWorkspaceRetentionConfig() {
     return new WorkspaceRetentionConfig(minDays, maxDays, maxSizeMb);
   }
 
+  @Override
+  public List<WorkerPodToleration> getWorkerPodTolerations() {
+    // Format:
```

Comment:
Nice comment.

Can this be a proper function doc string?

Reply:
Yeah sure